CODEOWNERS

```diff
@@ -399,12 +399,52 @@
 /samples/app_event_manager/               @nrfconnect/ncs-si-muffin @nrfconnect/ncs-si-bluebagel
 /samples/app_event_manager_profiler_tracer/ @nrfconnect/ncs-si-muffin @nrfconnect/ncs-si-bluebagel
 /samples/benchmarks/coremark/             @nrfconnect/ncs-si-bluebagel
-/samples/bluetooth/                       @nrfconnect/ncs-si-muffin @nrfconnect/ncs-dragoon
 /samples/bluetooth/broadcast_config_tool/ @nrfconnect/ncs-audio
-/samples/bluetooth/mesh/                  @nrfconnect/ncs-paladin
-/samples/bluetooth/direction_finding_connectionless_rx/ @ppryga-nordic
-/samples/bluetooth/direction_finding_connectionless_tx/ @ppryga-nordic
+/samples/bluetooth/central_and_peripheral_hr/ @nrfconnect/ncs-si-muffin
+/samples/bluetooth/central_bas/           @nrfconnect/ncs-si-muffin
+/samples/bluetooth/central_hids/          @nrfconnect/ncs-si-bluebagel
```

Comment:
@kapi-no just to make sure: do we take over BLE HIDS samples now ?

Reply:
yes